src/components/ProjectStatusIcon/index.jsx

```diff
@@ -10,13 +15,22 @@ import atoms from '../../atoms';
  * @returns {JSX.Element}
  */
 export default function ProjectStatusIcon({ id, size = 'md', className }) {
-  const status = atoms.useRecoilValue(
-    atoms.status(id),
-  );
+  const pubStatus = atoms.useRecoilValue(atoms.projectPublishedStatus(id));
+
+  const statusStyles = {
+    [STATUS_PROJECT_DRAFT]: null,
```

Comment:
Removing this wont have an effect on the styles right? Since it’ll return undefined anyway?